common/presentation/src/main/kotlin/com/pandulapeter/myTodos/presentation/ui/Demo2.kt

```diff
@@ -18,80 +20,66 @@ import com.pandulapeter.myTodos.model.Issue
 import com.pandulapeter.myTodos.presentation.resources.Dimension
 import com.pandulapeter.myTodos.presentation.resources.Text
 import com.pandulapeter.myTodos.presentation.utilities.get
-import kotlinx.coroutines.Dispatchers
-import kotlinx.coroutines.GlobalScope
-import kotlinx.coroutines.launch
 
 @Composable
-internal fun ColumnScope.Demo2(
-    getIssues: GetIssuesUseCase = get()
+internal fun Demo2(
+    getIssues: GetIssuesUseCase = get(),
 ) {
     var issues by remember { mutableStateOf<ResultWrapper<List<Issue>>?>(null) }
+    var retry by remember { mutableStateOf(Any()) }
 
-    fun loadIssues() {
-        issues = null
-        GlobalScope.launch {
-            val newIssues = getIssues()
-            launch(Dispatchers.Main) {
-                issues = newIssues
-            }
-        }
+    LaunchedEffect(getIssues, retry) {
```

Comment:
If you want to trigger a side effect when this composable enters the composition use [LaunchedEffect](https://developer.android.com/jetpack/compose/side-effects#launchedeffect) which conveniently exposes a coroutine scope. You already know this isn't an exception of using `GlobalScope` 😉.

In general you shouldn't rely on the Composable function being called to trigger side effects. If you need side effects use [SideEffect](https://developer.android.com/jetpack/compose/side-effects#sideeffect-publish)